vagrant/installer.sh

```diff
@@ -49,6 +49,19 @@ ansible-playbook -vvv -i ./inventory/hosts.ini configure-qat.yml | tee setup-qat
 
 # Kubernetes installation
 install_k8s
+if [[ "${CONTAINER_RUNTIME:-runc}" == "kata" ]]; then
+case ${ID,,} in
+    clear-linux-os)
+        sudo swupd bundle-add containers-virt
+        kubectl apply -f https://raw.githubusercontent.com/kata-containers/packaging/master/kata-deploy/kata-rbac.yaml
+        kubectl apply -f https://raw.githubusercontent.com/kata-containers/packaging/master/kata-deploy/kata-deploy.yaml
```

Comment:
The pre-built kata artifacts are not sufficient for the QAT usage. We will need the QAT VF drivers in Kata rootfs and the kernel with some ```CONFIG_``` options enabled.

Reply:
Got it, I'm going to include the Eric's instructions here. I've decided if should be better to have ansible playbook or bash script for this.

Reply:
Let's take those in a later PR.